infra/managed-identity/managed-identity.bicep

```diff
@@ -0,0 +1,37 @@
+@description('Optional. Name of the User Assigned Identity.')
+param managedIdentity object
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Required. Environment name.')
+param environment string
+
+@description('Optional. Date in the format yyyyMMdd-HHmmss.')
+param deploymentDate string = utcNow('yyyyMMdd-HHmmss')
+
+@description('Optional. Date in the format yyyy-MM-dd.')
+param createdDate string = utcNow('yyyy-MM-dd')
+
+var customTags = {
+  Location: location
+  CreatedDate: createdDate
+  Environment: environment
+}
+
+var defaultTags = union(json(loadTextContent('../default-tags.json')), customTags)
+
+var managedIdentityTags = {
```

Comment:
The purpose value here is not correct. I would like to understand the real purpose of this MI - what its used for, which part of the platform etc?